components/GameCard.js

```diff
@@ -1,27 +1,33 @@
 import Image from "next/image";
 import styled from "styled-components";
+import Link from "next/link";
+
+
 
 export default function GameCard({ game }) {
+  
   return (
+    <Link href={`/games/${game.id}`}>
```

Comment:
since this link contains more than plain text, please provide an "aria-label" in order to prevent screen reader to read out loud the entire content of this link.